.envTemplate

```diff
@@ -2,7 +2,7 @@ WES_ENDPOINT=http://localhost:8082/ga4gh/wes/v1
 DOCKER_ENGINE_URL=unix:///var/run/docker.sock
 #WES_SHARED_DIR_PROPERTY=
 SYNAPSE_USERNAME=
```

Comment:
Should references to SYNAPSE_USERNAME be removed, or do they still serve a purpose?

Reply:
That's a good question.  The Synapse credentials are used both to authenticate the Synapse client and the Docker client.  The Synapse client does not need the user name when a PAT is provided but the Docker client seems to make use of it:  The provided user name will appear in logs and be associated with the PAT in a credentials store. So I thought it best to leave the user name parameter in place.